### ISSUE TITLE
Assemblies are not marked as CLSCompliant

### ISSUE THREAD

**Issue by tiloc** [ryanhorath/PortableDataAnnotations#2]
The assemblies are not marked as CLSCompliant. This leads to compiler warnings if one references this library as part of another CLSCompliant library.

**Comment by soroshsabz** [comment]
ITNOA

@tiloc please create PR for CLSCompliant

**Comment by tiloc** [comment]
This library does not seem to be maintained anymore. Sorry, not seeing a point in submitting a PR if it won't get pulled.

**Comment by soroshsabz** [comment]
@tiloc yes you right ;) did you can any help about #4 ? I could resolve this problem


Thanks

**Comment by ryanhorath** [comment]
Sorry I did not respond sooner. I did pretty much abandon this project. I have been busy in my personal life and I believe .NET Standard addresses this issue and makes this library no longer necessary, though obviously that does not help people who are not yet on .NET Standard.

If you create a pull request I should be able to accept it. Pretty sure the code is compliant and all it will take is an assembly attribute. Sorry for the lack of response.
